From c2b97dd63f726fa3db7f699bb40b2be3e62b8df5 Mon Sep 17 00:00:00 2001
From: Kisaragi Hiu <mail@kisaragi-hiu.com>
Date: Fri, 31 Jan 2025 01:48:35 +0900
Subject: [PATCH] Apply suggestions from review
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

- Add ここを as per review
- Leave out unused strings from the expired-password branch

Co-authored-by: Jumpei Ogawa <phanective@gmail.com>
---
 data/translations/ja.ts | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/data/translations/ja.ts b/data/translations/ja.ts
index 0fb8ead7d..c113b8a8c 100644
--- a/data/translations/ja.ts
+++ b/data/translations/ja.ts
@@ -5,7 +5,7 @@
     <name>PictureBox</name>
     <message>
         <source>Press to login</source>
-        <translation>押してログイン</translation>
+        <translation>ここを押してログイン</translation>
     </message>
 </context>
 <context>
@@ -105,13 +105,5 @@
         <source>Hibernate</source>
         <translation>ハイバネート</translation>
     </message>
-    <message>
-        <source>Password change aborted because maximum tries reached</source>
-        <translation>最大試行回数に達したため、パスワード変更は中止しました</translation>
-    </message>
-    <message>
-        <source>New password change round! Please input current password again!</source>
-        <translation>新しいパスワード変更ラウンドです! 今のパスワードをもう一度入力してください!</translation>
-    </message>
 </context>
 </TS>
